compose/common.yml

```
@@ -64,8 +64,10 @@ x-provider-server-environment: &provider-server-environment
   ENABLE_METRICS: ${ENABLE_METRICS:-false}
   EPOCH_POLL_INTERVAL: ${EPOCH_POLL_INTERVAL:-500}
   SERVICE_NAMES: ${SERVICE_NAMES:-asset,chain-history,handle,network-info,rewards,stake-pool,tx-submit,utxo}
+  SUBMIT_API_URL: ${SUBMIT_API_URL:-http://cardano-submit-api:8090/api/submit/tx}
```

```suggestion
  SUBMIT_API_URL: ${SUBMIT_API_URL:-http://cardano-submit-api:8090}
```
No need to expose the internal path of the service, just the root URL

---

This is the `provider-server` configuration which contains the docker compose internal URL, nothing new is exposed.

---

Right, but it's better to just configure the root URL, then add `/api/submit/tx` inside the client

---

b6dc40411 and 81d6db18f should solve this